packages/hash/frontend/src/components/forms/OldButton.tsx

```diff
@@ -1,7 +1,7 @@
 import { forwardRef, ComponentProps } from "react";
 import { tw } from "twind";
 
```

Comment:
I'm not sure if this makes sense in react, but could add a `@deprecated` field for this